assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -199,7 +207,11 @@ function SizePositionPanel({
           suffix={__('Rotate', 'web-stories')}
           symbol={_x('°', 'Degrees, 0 - 360. ', 'web-stories')}
           value={rotationAngle}
-          onChange={(value) => pushUpdate({ rotationAngle: value })}
+          onChange={(value) =>
+            pushUpdate({
+              rotationAngle: isNum(value) ? value % 360 : value,
+            })
+          }
```

Comment:
Should a non-num value also be pushed upstream? Would that not make the element jump weirdly around on screen if typing something like `33.3` when you reach the decimal point?

Reply:
We've solved this elsewhere by controlling the value internally and only pushing valid values upstream (plus updating local value from upstream in a `useEffect`). However, we might get away with limiting this input completely to digits only and completely circumvent the need for an `isNum` check all together.

Reply:
Yeah. I think it does push it upstream here.